www/lobby/index.ts

```diff
@@ -0,0 +1,77 @@
+import { Router, Request, Response } from 'express';
+import { getClient } from '../../lib';
+import { Lobby } from '../../lib/lobby';
+import { COLLECTION } from '../../lib/private/enums';
+import { User } from '../../lib/user';
+import { generateLobbyId } from '../../utils/lobby';
+import { lobby_id_router } from './:id';
+
+export const lobby_router = Router();
+
+// verification middleware
+lobby_router.use('/midtest', async (req: Request, res: Response, next: any) => {
+  const userId = req.body.id;
+  const refreshToken = req.body.refreshToken;
+
+  const user = (await User.fromId(userId) ?? null);
+  if (!user) {
+    res.status(404).json('user DNE');
+    return;
+  }
+
+  const verify = await User.verifyRequest(userId, refreshToken);
+  if (verify.status !== 200) {
+    res.status(403).json('verify fail');
+    return;
+  }
+  next();
+});
+
+// Creates a new lobby and returns the lobby id w/ lobby data
+/**
+ * Body Params: lobbyName, theme, id, refreshToken
+ */
+lobby_router.post('/', async (req: Request, res: Response) => {
+  const lobbyName = req.body.lobbyName;
+  const theme = req.body.theme;
+  const userId = req.body.id;
+  const participants = [userId];
+
+  const manager = await User.fromId(userId) ?? null;
+  if (!manager) {
+    res.status(404).json('user dne');
+    return;
+  }
+
+  const client = await getClient();
+  let retryNum = 0;
+  let lobbyId = generateLobbyId(userId, retryNum).toString();
```

Comment:
if u are having to do `toString()` all the time, it means there is something wrong with ur function!

```suggestion
  let lobbyId = generateLobbyId(userId, retryNum);
```